src/pages/patterns/buttons.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Buttons
+description: >
+  Know why they're pushing your buttons.
+---
+
+## Overview
+
+Buttons are common and vital elements because they help make content interactive across the
+internet. They allow a user to take actions with a single click and are placed throughout a
+page or workflow. Buttons use visual signifiers such as shape, color, and size to show that
+a user can interact with them.
+
+## Interface
+
+**Primary buttons** are for the most important action in a given view. There is typically
+one call-to-action on a page or view. **Default** buttons are for secondary actions.
+**Basic buttons** are for repetitive or deemphasized actions in a given view.
+
+![Tone matrix map](abstract:patterns-buttons-type-usage.png)
+
+## Size
+
+The default button size is medium. In screens that have more information density, use small
+buttons. Large buttons are mostly used in onboarding. Buttons and text inputs within the same
+flow should be the same size.
+
+<BestPractice>
+  <Do imageSource={props.data.buttonsSizeDo.childFile.childImageSharp.fluid} />
+  <Dont imageSource={props.data.buttonsSizeDont.childFile.childImageSharp.fluid} />
+</BestPractice>
+
+## Content
+
+Use clear, action-oriented language to help people choose the right action for them. A typical
+construction is `Verb noun`. It’s OK and expected for Button labels to mirror the content of a
+page or modal heading.
+
+Lead with verbs and omit articles, pronouns, and punctuation. Avoid terse labels that need context
+like "Yes," "No," or "Done." Buttons are sentence case, not title case or all caps. Use 1–4 words
+and a maximum of 20 characters in English including spaces to adhere to internationalization guidelines.
+
+<BestPractice>
+  <Do imageSource={props.data.buttonsContentVerbDo.childFile.childImageSharp.fluid}>
+    <ul>
+      <li>Use labels specific to the action they perform</li>
+      <li>Communicate a single action per button</li>
+    </ul>
+  </Do>
+  <Dont imageSource={props.data.buttonsContentVerbDont.childFile.childImageSharp.fluid}>
+    <ul>
+      <li>Use general button labels that might cause confusion</li>
+      <li>Use labels that need more context</li>
```

Comment:
The mismatch of putting "Use" imperative text under a "Don't" category is confusing. What am I actually supposed to do?